swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
+from uuid import uuid4
+
+CONDITION_DESCRIPTION = { 0: "heavily used", 
+                        1: "used", 
+                        2: "fair", 
+                        3: "good", 
+                        4: "like new", 
+                        5: "mint"}
+
+
+#wave 2
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        self.id = uuid4().int if id is None else id
+        self.condition = condition
+
+
+    def get_category(self):
+        return "Item" #returnig class as a string
+
+
+#wave 3 
+    def __str__(self):
+        return f"An object of type Item with id {self.id}." # same as get_category, there maybe hard code issue
```

Comment:
We don't need to hardcode the word "Item" in this sentence because we have the `get_category` function. 

If you dynamically call `get_category` then `__str__` becomes more flexible and the child classes of `Item` can also use `__str__` instead of repeating code. 

```suggestion
        return f"An object of type {self.get_category} with id {self.id}." # same as get_category, there maybe hard code issue
```